tests/test_scripts.py

```diff
@@ -591,6 +591,462 @@ def test_filter_stoptags_fq():
     assert 'seq 1::BAR' in names
 
 
+def test_normalize_by_median_indent():
```

Comment:
@luizirber , is this block of added tests an extraneous inclusion? I didn't write them...